pkg/iac/terraform/state/terraform_state_reader_acc_test.go

```diff
@@ -20,7 +17,7 @@ func TestAcc_StateReader_WithMultipleStatesInDirectory(t *testing.T) {
 		Args: []string{
 			"scan",
 			"--from", "tfstate://testdata/acc/multiple_states_local/states",
-			"--filter", "Type=='aws_s3_bucket' || Type=='aws_route53_zone'",
+			"--filter", "(Type=='aws_s3_bucket' && Id != 'aws-cloudtrail-logs-994475276861-f6865496') || Type=='aws_route53_zone'",
```

Comment:
Any context about that filter ? What is `aws-cloudtrail-logs-994475276861-f6865496` ?

Reply:
it's an internal bucket for cloudtrail in our QA account that we want to ignore

Reply:
May be we could use a prefix instead (`starts_with(Id, \'aws-cloudtrail-logs-\')`), using the full id might lead to unexpected failures - WDYT?